test/test-v8debugapi.ts

```diff
@@ -500,6 +500,29 @@ describe('v8debugapi', function() {
 
     });
 
+    it('should resolve correct breakpoint line number', function(done) {
```

Comment:
Could the test name be updated to mention why the line number may have been incorrect?